src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -94,38 +98,31 @@ private Document createXmlDoc(Schema schema, Object value, String rootName) {
                 for (final Field field : schema.fields()) {
                     processSchema(doc, schemaSequence, field.name(), field.schema());
                 }
-            }
-            else if (schema.type() == Type.ARRAY) {
+            } else if (schema.type() == Type.ARRAY) {
                 processSchema(doc, schemaSequence, "entry", schema);
-            }
-            else if (schema.type() == Type.MAP) {
+            } else if (schema.type() == Type.MAP) {
                 processMapSchema(doc, schemaSequence, schema);
-            }
-            else {
+            } else {
                 log.error("Unknown type found in schema {}", schema.type().getName());
             }
 
             root.appendChild(schemaElement);
         }
 
         if (value instanceof Struct) {
-            processStruct(doc, root, (Struct)value);
-        }
-        else if (value instanceof Collection) {
-            addListElements(doc, root, (Collection<?>)value, "entry");
-        }
-        else if (value instanceof Map) {
-            processMap(doc, root, (Map<?, ?>)value);
-        }
-        else {
+            processStruct(doc, root, (Struct) value);
+        } else if (value instanceof Collection) {
+            addListElements(doc, root, (Collection<?>) value, "entry");
+        } else if (value instanceof Map) {
```

Comment:
```suggestion
        } 
        else if (value instanceof Map) {
```

Unrelated to this specific PR